loadtest/user/userentity/actions.go

```diff
@@ -694,8 +694,11 @@ func (ue *UserEntity) GetTeamMembersForUser(userId string) error {
 
 // GetUsersByIds fetches and stores the specified users.
 // It returns a list of those users' ids.
-func (ue *UserEntity) GetUsersByIds(userIds []string) ([]string, error) {
-	users, _, err := ue.client.GetUsersByIds(context.Background(), userIds)
+func (ue *UserEntity) GetUsersByIds(userIds []string, since int64) ([]string, error) {
+	opts := &model.UserGetByIdsOptions{
+		Since: since,
+	}
+	users, _, err := ue.client.GetUsersByIdsWithOptions(context.Background(), userIds, opts)
```

Comment:
It seems that the server code applies the `u.UpdateAt > since` in the SQL query if the query parameter `since` is non-empty, even if it equals "0". Should we treat `since = 0` here as a special case and avoid passing the options? Something like (not tested):
```suggestion
func (ue *UserEntity) GetUsersByIds(userIds []string, since int64) ([]string, error) {
	opts :=  &model.UserGetByIdsOptions{}
	if since > 0 {
		opts.Since = since
	}
	users, _, err := ue.client.GetUsersByIdsWithOptions(context.Background(), userIds, opts)
```

Reply:
No need. See the `(c *Client4) GetUsersByIdsWithOptions` code. It only sets the query param, if `since` is non-empty.  Which automatically does what you want.

Reply:
Aaah, went straight to the server code, I had not checked the client code. Good then!